core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -20,8 +20,9 @@ package org.apache.spark
 import java.io._
 import java.util.concurrent.{ConcurrentHashMap, LinkedBlockingQueue, ThreadPoolExecutor, TimeUnit}
 import java.util.concurrent.locks.ReentrantReadWriteLock
-import java.util.zip.{GZIPInputStream, GZIPOutputStream}
 
+import com.github.luben.zstd.ZstdInputStream
+import com.github.luben.zstd.ZstdOutputStream
```

Comment:
IIRC, these imports should fall into third party imports, thus under the scala imports according to the coding style.

Reply:
Thanks. Intellji always moves my import around when I do auto-import. Going to turn off that.